src/components/QuranReader/ReadingView/Page.tsx

```diff
@@ -12,11 +15,16 @@ type PageProps = {
 
 const Page = ({ verses, page }: PageProps) => {
   const lines = useMemo(() => groupLinesByVerses(verses), [verses]);
+  const { quranTextFontScale } = useSelector(selectQuranReaderStyles) as QuranReaderStyles;
+  const hasSpecialLayout = quranTextFontScale > 3;
```

Comment:
should we call this bigText to be consistent with our usage in the previous PR. We can call it bigTextLayout. Special is a bit vague, open to other suggestions.